samples/cpp/benchmark_app/utils.cpp

```diff
@@ -759,7 +759,19 @@ void load_config(const std::string& filename, std::map<std::string, ov::AnyMap>&
         }
         for (auto iit = device.begin(); iit != device.end(); ++iit) {
             auto item = *iit;
-            config[opencv_to_plugin_format(device.name())][item.name()] = item.string();
+            if (item.string().find(" ") == std::string::npos) {
```

Comment:
I think it is good time to add simple sample how config for benchmark_app should looks like. At least describe it in README

Reply:
Updated.